Source/WSTagView.swift

```diff
@@ -111,6 +155,14 @@ open class WSTagView: UIView, UITextInputTraits {
         textColor = .white
         selectedColor = .gray
         selectedTextColor = .black
+        
+        imageView.frame = CGRect(x: layoutMargins.left, y: layoutMargins.top, width: imageSize.width, height: imageSize.height)
+        imageView.layer.cornerRadius = 10
+        imageView.isHidden = isImageHidden
+        imageView.image = imagePlaceholder
+        imageView.tintColor = .white
+        imageView.backgroundColor = .clear
```

Comment:
If the image is greater than 22,22, we might need to tell it to resize automatically